src/formats/rec.c

```diff
@@ -41,6 +41,12 @@ void sd_rec_free(sd_rec_file *rec) {
         }
         omf_free(rec->moves);
     }
+
+    for(int i = 0; i < 2; i++) {
+        if(rec->pilots[i].has_photo) {
+            sd_sprite_free(&rec->pilots[i].photo);
```

Comment:
sd_pilot_free() ?

Reply:
but we don't allocate a pilot... we allocate a sprite on line 80.

Reply:
oh, I guess we do, idk, this fixes the leak??

Reply:
sd_pilot_free also does omf_free() for the photo, should that also be done here ?

Reply:
the photo isn't even in the sd_pilot struct though